assets/queries/cloudFormation/iot_policy_allows_resource/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "iot_policy_allows_resource",
```

Comment:
Change query directory name to: `iot_policy_allows_wildcard_resource`
Change query id to: `iot_policy_allows_wildcard_resource`
Change query name to: `IoT Policy Allows Wildcard Resource`